internal/app/config.go

```diff
@@ -46,7 +46,7 @@ type Auth0Config struct {
 }
 
 type AuthSrvConfig struct {
-	Domain string `default:"http://localhost:8080"`
+	Domain string `default:"https://localhost:8080"`
```

Comment:
```suggestion
	Domain string `default:"http://localhost:8080"`
```

HTTPS is not suitable for local servers because it requires self-signed certificates

Reply:
sorry it's forgotten 
I can't run the server locally without HTTPS

Reply:
Could you please elaborate on what the situation is?